assets/extensions/all-extensions.js

```diff
@@ -60,31 +60,35 @@ const renderSections = ( layout, extensionsBySlug ) =>
 							itemProps = {},
 							wrapperProps = {},
 							cardProps = {},
-						} ) => (
-							<li
-								{ ...itemProps }
-								key={ key }
-								className={ classnames(
-									'sensei-extensions__list-item',
-									itemProps?.className
-								) }
-							>
-								<div
-									{ ...wrapperProps }
+						} ) =>
+							( ! extensionSlug ||
```

Comment:
This diff got a little confusing because it changed the indentation. Basically, it introduces the `( ! extensionSlug || extensionsBySlug[ extensionSlug ] )` check to avoid listing extensions that we don't have. It's for cases that we remove extensions, and it's still in the layout, for example.